### DISCUSSION TITLE
bug: Download failed  Sadly we failed at downloading that specific media. We try to support as many websites as possible, so it would help a lot if you could report that error!

### DISCUSSION THREAD

**Discussion by adilgit88** [aclap-dev/video-downloadhelper#552] [Help and Q&A]
### Your question

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://www.udemy.com/course/ios-13-app-development-bootcamp/learn/lecture/16252786#overview",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [in#0 @ 0x154f08f20] Error opening input: Server returned 403 Forbidden (access denied)\nError opening input file https://www.udemy.com/assets/19926152/files/2019-09-13_17-09-51-4a18b69360f2e2eb82f06a32d5d7bb95/2/hls/AVC_1920x1080_1200k_AAC-HE_64k/aa004387df33dcc438e2dc0758a22e97034e.m3u8?provider=cloudfront&token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJwYXRoIjoiMjAxOS0wOS0xM18xNy0wOS01MS00YTE4YjY5MzYwZjJlMmViODJmMDZhMzJkNWQ3YmI5NS8yLyIsImV4cCI6MTcxNjQzMzA2NH0.IBunzY8EscCWkZhQoZejoRXaBCwemZ1ZU6IQjAYoX_Y&v=1.\nError opening input files: Server returned 403 Forbidden (access denied)\n",
    "id": "coapp_failure_2c24e050-25a8-489f-a882-5cbc836b1616",
    "downloadable_id": "downloadable_1647235819088381"
  }
}

need help for fit it. 

### Addon & CoApp version and details

need help for fix the bug

**Comment by Smietje77** [comment]
{
  "version": "9.0.1.8",
  "target": "google",
  "channel": "beta",
  "page_url": "https://www.udemy.com/course/build-your-sales-funnel-with-chatgpt/learn/lecture/38352438#overview",
  "error": {
    "details": "SideDownload error: : [mov,mp4,m4a,3gp,3g2,mj2 @ 000002c8c42aa500] trun track id unknown, no tfhd was found\r\n[mov,mp4,m4a,3gp,3g2,mj2 @ 000002c8c42aa500] error reading header\r\n[in#0 @ 000002c8c2ae8e40] Error opening input: Invalid data found when processing input\r\nError opening input file https://dash-enc-c.udemycdn.com/out/v1/2d72e2fff73d4b028b8f57406187e4ca/c2b2b17f0c7c4ee1b0624cfb7d543d6a/41ad744bac0b4bd88239b4389364379c/d7d8cdfe674f45c3a2cbac812a02ac35/index_video_11_0_6.mp4?Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cHM6Ly9kYXNoLWVuYy1jLnVkZW15Y2RuLmNvbS9vdXQvdjEvMmQ3MmUyZmZmNzNkNGIwMjhiOGY1NzQwNjE4N2U0Y2EvYzJiMmIxN2YwYzdjNGVlMWIwNjI0Y2ZiN2Q1NDNkNmEvNDFhZDc0NGJhYzBiNGJkODgyMzliNDM4OTM2NDM3OWMvZDdkOGNkZmU2NzRmNDVjM2EyY2JhYzgxMmEwMmFjMzUvKiIsIkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MTcxNzI3MDQ1N319fV19&Signature=pMf7BM15hqhfu1X-LBJQbpivENfpuZr9loooGST9B3mSxzV9PO9UnpC1uDAQUZwid19wX5uqW5JraG1P7ey3BVo8ZPl4ma8ljBcYI~7M6MCWuboNxdsXsfaRoDwig6N-YrbkItly-QH-K1f8OfEhxU1eaJMx7GimqBx0qpJ0n7EEsL7jRnw7lbobGsCfUYto~KcqLzYO6H3p2UXfWnMdSI5ZvLO1zigl9OVFAdTncE1iqzVLQ-WreWtRm~thBBW-HeWga4atzRU6oZfmLSOCf2hV93ApcMTCWdbVLYbGx~Qeb6CEogxRx-sbHCflBndbW8f1KsPtOYEPEzUKXuSiTA__&Key-Pair-Id=K3MG148K9RIRF4&CMCD=bl%3D15200%2Cbr%3D775%2Cd%3D6000%2Cdl%3D15200%2Cmtp%3D56200%2Cnor%3D%22index_video_11_0_7.mp4%22%2Cot%3Dv%2Crtp%3D1500%2Csf%3Dd%2Csid%3D%22e607deb8-b10f-4ea8-b04b-1740953ee8f5%22%2Cst%3Dv%2Ctb%3D775.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_6716943714172062",
    "error": "coapp_failure",
    "id": "coapp_failure_89ee866e-fc61-4de5-9f6c-4ffc4701fff5"
  }
}

**Comment by GrampaWildWilly** [comment]
Refer to #761.
